runtimeconfig/google/cloud/runtimeconfig/_helpers.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Shared helper functions for RuntimeConfig API classes."""
+
+
+def _config_name_from_full_name(full_name):
+    """Extract the config name from a full resource name.
+    
+      >>> _config_name_from_full_name('projects/my-proj/configs/my-config')
+      "my-config"
+    """
+    return full_name.split('/')[3]
```

Comment:
This is pretty brittle. If you want to assert there are 4 parts you should do the following:

```python
_, _, _, result = full_name.split('/')
return result
```

But it's unclear that there would also be 4 parts. You may be happier just using

```python
separator = '/'
if separator in full_name:
    _, result = full_name.rsplit(separator, 1)
else:
    result = full_name
```